telegraf/1.18/alpine/entrypoint.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -e
 
+# Allow telegraf to send ICMP packets and bind to privliged ports
+setcap cap_net_raw,cap_net_bind_service+ep /usr/bin/telegraf
```

Comment:
Do we need to add `apk add libcap` to the dockerfile to make sure setcap is available here? Explicitly installing it isn't so important when we call it in the dockerfile because if it was missing there the build would fail. If it's missing here the failure is at runtime.

It's definitely in the alpine base we're currently using. This would only matter if a future base image doesn't have it.

Reply:
I can defensively add it. I like to assume that something like this would not get removed given its wide use.